tests/test_main.py

```diff
@@ -314,14 +321,18 @@ def test_export_bulk_csv(requests_mock, tmp_path):
 
 
 def test_export_bulk(moodle_requests_mock, tmp_path, mocker):
+    random.seed(1)
     runner = CliRunner()
 
     s3_client = boto3.client('s3')
     stubber = botocore.stub.Stubber(s3_client)
 
     directory = 'path'
     bucket_name = 'test-bucket'
-    data = {'courseid': 21, 'users': {'id': 3}}
+    grade_data = {'courseid': 21, 'users': {'id': 3}}
+    generated_uuid = str(UUID('cd613e30d8f16adf91b7584a2265b1f5'))
+    user_data = [{'id': 2, 'user_uuid': 'abcd'},
+                 {'id': 3, 'user_uuid': generated_uuid}]
```

Comment:
Depending on whether we want to put `null` or keep the object sparse, the expected data can be updated here.